.gitignore

```diff
@@ -6,3 +6,5 @@ pkg
 
 .gems
 .rbenv-gemsets
+
+Gemfile.lock
```

Comment:
I prefer keeping it to make contribution more stable https://grosser.it/2015/08/14/check-in-your-gemfile-lock/